Android.mk

```diff
@@ -43,6 +43,7 @@ srcs +=	adbg/src/adbg_case.c \
 	adbg/src/adbg_run.c \
 	adbg/src/security_utils_hex.c \
 	aes_perf.c \
+	acipher_perf.c \
```

Comment:
s/acipher_perf.c/asym_perf.c/ if you change from `--acipher-perf` to `--asym-perf`